package.json

```diff
@@ -1,87 +1,93 @@
 {
   "name": "react-oui-icons",
-  "version": "1.0.1",
-  "main": "lib/react-oui-icons.js",
+  "version": "2.0.0",
+  "main": "index.js",
+  "module": "src/index.js",
   "private": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/optimizely/react-oui-icons.git"
+  },
+  "keywords": [
+    "oui",
+    "icons",
+    "components",
+    "optimizely",
+    "react",
+    "ui"
+  ],
+  "author": "Iñaki Aranzadi <inaki.aranzadi@optimizely.com> ",
+  "contributors": [
+    {
+      "name": "Dave Rau",
+      "email": "dave.rau@optimizely.com"
+    },
+    {
+      "name": "James Fox",
+      "email": "james@optimizely.com"
+    },
+    {
+      "name": "Michael Fields",
+      "email": "michael.fields@optimizely.com"
+    }
+  ],
+  "license": "ISC",
+  "scripts": {
+    "build": "rm -Rf ./build && mkdir build && npx babel ./src --ignore index.test.js --out-file ./build/index.js --source-maps && cp ./package.json ./build/package.json && cp -R ./src ./build/src",
```

Comment:
What is this last part needed for? (copying package.json and /src to the build directory)?

`&& cp ./package.json ./build/package.json && cp -R ./src ./build/src"`

Reply:
I'm assuming that we'll start using the `npm publish ./build` command which will publish everything in the ./build folder to npm.  We need a package.json in the published module so I'm copying the top-level one into the build folder.  And the copy of the source similarly so that the full source is included in the npm distribution.

Reply:
I dont think we need to restrict it to just that folder (https://docs.npmjs.com/cli/publish)

I think its easier to just use the root packge.json and control what gets exported either with `.npmignore` (blacklist) or the `package.files` property (whitelist)

Reply:
For this case I think its simplest to do:

```
  main: 'build/index.js',
  module: 'src/index.js',
  files: ['src', 'build']
  build: `npx babel ./src --ignore index.test.js --out-file ./build/index.js --source-map`
```

Then travis just runs:
```
npm run build
npm publish
```